### DISCUSSION TITLE
[HELP] Hosting example wiki

### DISCUSSION THREAD

**Discussion by paLeziart** [gollum/gollum#2053] [Q&A] [answered]
Hello,

This is not a bug report or a demand for feature, but rather a small question related to a part of the documentation that I might have missed. Basically: how to host the example lotr.git wiki? I am just starting out with Gollum and I wanted to have some base. I saw that you had an example in the test directory and tried hosting it with `gollum . --host localhost` while being inside the `lotr.git` directory.

I obtain the following result, which I guess is not the expected output because I am seeing a whole tree of pages in "Overview" that do not appear.

![gollum](https://github.com/gollum/gollum/assets/47817000/d2b2514d-02ee-4d9f-9967-6e15950dac5d)

Note that I had to install the RedCloth gem which was not in the requirements. I had a rendering error message about a missing `redcloth` renderer.

Is there a particular way to host this example wiki? Or maybe another wiki I could start with?

Thank you for your help!

**Environment Info**
I did a system-wide installation on Linux Mint 21.3.
```
Gollum 6.0.0
Running on: x86_64-linux-gnu with Ruby version 3.0.2
Using:
rugged 1.7.2
gollum-rugged_adapter 3.0
gollum-lib 6.0
Markdown rendering gem: kramdown
Other rendering gems:
RedCloth 4.3.4
```

**Comment by benjaminwil** [comment, marked as answer]
Hey, I have just followed what you did to see what you mean, and I think you are seeing the expected result just fine.

I did this:

```sh
gem install gollum -v 5.3.3
gem install RedCloth
cd my-local-path-to-lotr.git
gollum . --host localhost
```

I see exactly what you see on the home page. And the overview page lists all of the files checked into the LOTR example git repo. To my knowledge, this is all the LOTR wiki is. Off hand, I am unsure whether there would be a more filled out example wiki to start with, but any git repository full of Markdown files could make a good starting point.

What do you feel is missing? Or is there some specific functionality you're looking to test?

**Reply by paLeziart** [reply]
Thank you for your answer! 

I was expecting something to see something like [the Screenshots page](https://github.com/gollum/gollum/wiki/Screenshots#general) where you have a Bilbo Baggings page with some example text, a math equation and a side TOC.

![bilbo](https://github.com/gollum/gollum/assets/47817000/bf0a7b80-4ae3-44e9-a453-2b23968e4f82)

I got confused because it starts with a single word "Hobbits", which is also the single starting word on the Home page I got. So I thought I was not seeing the rest because of some broken rendering on my side. Moreover, the Bilbo Baggins was "broken" (red) although the page seemed to exist, so it reinforced my sentiment something was amiss with this bare-bone home page.

Sorry for the noise :sweat_smile:

**Reply by benjaminwil** [reply]
Not noise at all! I find it valuable to understand what things could be better for users just getting started. 👍

**Comment by benjaminwil** [comment]
> Note that I had to install the RedCloth gem which was not in the requirements. I had a rendering error message about a missing `redcloth` renderer.

I think this is a bug. Gollum should use the default Markdown renderer and it shouldn't matter that `RedCloth` is not installed. I will make an issue to track this if there isn't one already.

**Reply by dometto** [reply]
The issue is that `redcloth` is a renderer for textile markup files and not for markdown. And for some reason the LOTR repo uses a textile and not a markdown file for `Home`.. Basically LOTR is really only meant as a test repository, so it has many different markups in there for the test suite. If you want to give gollum a spin, maybe cloning the official documentation wiki is better? That's at:
 https://github.com/gollum/gollum/wiki.git

This ties in with a longstanding issue, which is that we should really provide an online sandbox/test instance for people to try out gollum.
